2-python-fundamentals-i.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `2-python-fundamentals-i.ipynb` Cell 137 Line 4_
```markdown
**Lists** are the most important and common data type in Python, so understanding their construction and manipulation is important. 

The following are helpful resources about lists:
* [Python tutorial on lists](https://docs.python.org/3/tutorial/introduction.html#lists)
```
Good call on linking out to trusted resources, that can be hard to navigate for newcomers, I'll be sure to add that to my modules